WafflePedia/.github/workflows/custom_config.yml

```
@@ -0,0 +1,58 @@
+name: CI-CD pipeline to AWS
+env:
+  EB_S3_BUCKET_NAME: ["elasticbeanstalk-ap-northeast-2-706612987643"]
+  EB_APPLICATION_NAME: ["wafflepedia"]
+  EB_ENVIRONMENT_NAME: ["WafflePedia-env"] # 보통 앱 이름에 -env 형태로 자동 생성
+  DEPLOY_PACKAGE_NAME: "django-app-${{ github.sha }}.zip"
+  AWS_REGION_NAME: ["ap-northeast-2"] # ex) ap-northeast-2
+
+on:
+  push:
+    branches:
+        - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git clone on our repo
+        uses: actions/checkout@v2
+
+      - name: Create zip deployment package
+        run: zip -r ${{ env.DEPLOY_PACKAGE_NAME }} ./ -x *.git*
```

제가 엘라스틱 빈스톡을 많이 안 써보긴 했는데요..
이렇게 되면 `WafflePedia` 디렉토리가 아니라 프로젝트 루트디렉토리가 압축될 것 같은데 그러면 [WSGIPath](https://github.com/wafflestudio21-5/team6-server/blob/e2ff8e5dc7361a33af3c15a1a1b3f0c4a41007c7/WafflePedia/.ebextensions/django.config#L3) 가 `WafflePedia.WafflePedia.wsgi:application` 이 되어야 하지 않을까요??
아니면 이 라인을 이렇게 바꾸어야할 것 같습니다.
```suggestion
        run: zip -r ${{ env.DEPLOY_PACKAGE_NAME }} ./WafflePedia -x *.git*
```
그리고 사실 `./WafflePedia` 경로가 좀 불필요하게 중첩되어 있는 것 같은 느껴집니다. 한 꺼풀 벗겨냈어도 되지 않나 싶어요.